webrisk/database_test.go

```diff
@@ -0,0 +1,753 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package webrisk
+
+import (
+	"context"
+	"encoding/hex"
+	"errors"
+	"io"
+	"io/ioutil"
+	"log"
+	"math"
+	"os"
+	"reflect"
+	"sort"
+	"strings"
+	"testing"
+	"time"
+	"fmt"
+
+	pb "github.com/GoogleCloudPlatform/golang-samples/webrisk/internal/webrisk_proto"
+	pt "github.com/golang/protobuf/ptypes"
+)
+
+func mustGetTempFile(t *testing.T) string {
+	f, err := ioutil.TempFile("", "")
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
```

Comment:
Call out the function that failed in test failures.

https://github.com/golang/go/wiki/CodeReviewComments#useful-test-failures

Reply:
Done.